pkg/config/config_template.yaml

```diff
@@ -306,6 +306,30 @@ metadata_collectors:
 #
 {{ end -}}
 
+# Datadog Cluster Agent - DCA
+#
+# The Authentication Token used to query the Datadog cluster agent from the node agent.
+# This token is provided in the header "Authorization" of each request, its length be must
+# greater than 32 characters.
+# If the value of cluster_agent_auth_token is left unset, the agent will fetch the content of
+# the 'dca_auth_token' file. This file must be present in the same directory as datadog.yaml.
+# cluster_agent_auth_token: ""
+#
+#
+# Cluster agent connectivity
+#
+# The cluster agent is a remote service, you need to provide an endpoint to query it.
+# The only supported scheme to reach the Datadog cluster agent is https, example: "https://host:port"
+# cluster_agent_url: ""
+
+# If the node-agent runs inside a Kubernetes Pod, you can use a Kubernetes Service to discover the
+# cluster agent.
+# To select this configuration, you need to leave the cluster_agent_url unset ("") and provide the
+# name of the Kubernetes service.
+# cluster_agent_kubernetes_service_name: "dca"
+#
+# The cluster_agent_url is evaluated before cluster_agent_kubernetes_service_name.
```

Comment:
this also sounds like an implementation detail (the sentence above, `To select this configuration [...]`, is a bit redundant with this one)